faq.md

```diff
@@ -1,112 +1,110 @@
-- La modération se réserve le droit de supprimer un message si elle juge le projet trop borderline, tendancieux, dangereux, ou pas assez sérieux.
-- Aucune discussion ne sera tolérée sur le salon
-- Ne pas reposter votre message avant un minimum de 3 mois, et uniquement en cas d'avancée majeure
+Si jamais votre code est beaucoup trop long pour tenir en un message ou que vous remplissez votre écran avec, il existe des sites pour poster vos codes. Il vous suffit de copier/coller votre code dans le site et de le sauvegarder, vous serez redirigés sur une page où votre code a été sauvegardé et vous pourrez copier/coller l'url unique ici sur discord.
 
-### Pourquoi mon sujet de conversation a été stoppé par la modération ?
-Dans le cas ou un(e) modérateur(trice) décide de couper court à votre conversation, c'est que cette dernière ne suit pas l'idéologie du serveur ou est
-simplement considérée trop borderline (à risque concernant les règles ou les lois) pour avoir lieu sur le serveur. En effet, personne ne peut prédire
-qui lit la conversation. Si le serveur donne de l'aide sur une forme d'attaque en sécurité, et ce même si la conversation est orientée sur la légalité, un
-membre qui lui n'a pas des intentions honnêtes pourrait se servir des indications données. C'est l'une des raisons pour lesquelles les sujets
-borderlines sont prohibés.
+- <https://paste.artemix.org/>
+- <https://bin.readthedocs.fr/>
 
-### Je trouve que la modération s'acharne sur mon compte, que dois-je faire ?
-L'équipe de modération voue son temps libre à maintenir un ordre et le sérieux sur le serveur. La modération ne s'acharne pas sur vous mais sur votre comportement,
-ainsi, si vous voulez éviter tout problème ou simplement d'être dans le viseur de l'équipe, suivez simplement les règles qui se veulent le moins
-restrictives possibles.
+### Demander de l'aide dans plusieurs salons (`!crosspost`)
 
-### Pouvez-vous ajouter un salon pour la technologie X ?
-Non, on ne crée pas de salon pour que des gens parlent peut-être dedans, ou pour tenter de créer une activité sur une thématique définie. Les salons sont créés quand il y a une forte demande, des discussions fréquentes, et un BESOIN de les isoler parce que ça nuit à d'autres salons.
+Normalement, les salons sont correctements découpés pour éviter qu'une même question soit posée à plusieurs endroits en même temps. Il se peut que vous vous trompiez de section et qu'un membre vous redirige ailleurs, nous vous prions de bien vouloir supprimer votre message de l'endroit original où vous l'aviez posté pour le reposter dans le bon salon. Pensez à prévenir qu'on vous a redirigé. Il n'y a pas de raison qui justifie qu'une question soit posée dans deux salons différents en même temps.
 
-## Demandes d'aides
+Si vous posez votre question sur plusieurs serveurs discord et que vous recevez une réponse sur un autre serveur, merci de revenir sur NaN pour prévenir que le problème est résolu.
```

Comment:
Je pinaille un peu, mais j'aurais mis Discord avec une majuscule ici.

Reply:
perso j'aurais mit Discord partout avec des majuscules

Reply:
Rhalala, suggestion ! ainsi j'ai juste à cliquer sur "add suggestion to commit" :p